to.etc.domui/src/main/java/to/etc/domui/component/wizard/WizardPopupStepNavbarFragment.java

```diff
@@ -0,0 +1,39 @@
+package to.etc.domui.component.wizard;
+
+import to.etc.domui.component.buttons.*;
+import to.etc.domui.dom.html.*;
+
+import javax.annotation.*;
+
+/**
+ * Holds the fragment for the navbar of the {@link WizardPopupWindow}
+ * @author <a href="mailto:yoeri.nijs@itris.nl">Yoeri Nijs</a>
+ * Created on 2-6-17.
+ */
+@DefaultNonNull
+public class WizardPopupStepNavbarFragment extends Div {
+
+	private String m_name;
+
+	private boolean m_current;
+
+	WizardPopupStepNavbarFragment(@Nonnull WizardPopupStep step, boolean isCurrentStep) {
```

Comment:
Being the current step should not be a property of this class, and also not be a part of the constructor, since who is current changes.
Having current in this class means that when the current thing changes you need to walk all other instances, setting current to false, before setting this to true.

The "current step" should be a property of the main wizard class, and by binding that property to this instance you can make the updating automatic.